### DISCUSSION TITLE
Network issue after upgrading to v3.2.4

### DISCUSSION THREAD

**Discussion by blake256** [utmapp/UTM#4054] [UTM Help (macOS)] [answered]
After upgrading from UTM v3.1.5 to v3.2.4 the network does not work anymore.

Configuration

UTM Version: 3.2.4
OS Version: Ubuntu 20.04 LTS arm64
Intel or Apple Silicon? Silicon

**Comment by osy** [comment, marked as answer]
Please try the troubleshooting tip from https://mac.getutm.app/gallery/ubuntu-20-04
